Gemfile

```diff
@@ -30,6 +28,8 @@ group :development do
   gem "puppet-module-posix-dev-r#{minor_version}", '~> 0.4',     require: false, platforms: [:ruby]
   gem "puppet-module-win-default-r#{minor_version}", '~> 0.4',   require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "puppet-module-win-dev-r#{minor_version}", '~> 0.4',       require: false, platforms: [:mswin, :mingw, :x64_mingw]
+  gem "gem",                                                     require: false
```

Comment:
This looks strange, is it supposed to be there? :-) @david22swan

Reply:
It is repeated in the line below also.

Reply:
Removed